orttraining/orttraining/test/training_ops/cpu/nn/dropout_op_test.cc

```diff
@@ -34,7 +34,7 @@ const Tensor& FetchTensor(const OrtValue& ort_value) {
 }
 
 void RunDropoutTest(const char* op, const bool use_mask, const std::vector<int64_t>& input_shape, float ratio = -1,
```

Comment:
See line 76. 
it's already using counting 0s and compare it with the ratio.

Reply:
I’m aware but what is your point? These are ORT tests not ONNX tests.

Reply:
You can enrich output_verifier if needed. 
But modifying optest and comparer utility is too intrusive.